src/distilabel/pipeline/_dag.py

```diff
@@ -749,3 +750,141 @@ def from_dict(cls, data: Dict[str, Any]) -> "DAG":
             )
 
         return dag
+
+    def _get_graph_info_for_draw(self) -> Dict[str, Any]:
```

Comment:
This seems to return a tuple instead of a dict. Also, could the docstring inform of what each variable is briefly?

Reply:
updated